src/databricks/sqlalchemy/_information_schema.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import MetaData, Table, Column, String, DateTime
+
+metadata = MetaData()
+
+tables = Table(
+    "tables",
+    metadata,
+    Column("table_catalog", String),
+    Column("table_schema", String),
+    Column("table_name", String),
+    Column("table_type", String),
+    Column("is_insertable_into", String),
+    Column("commit_action", String),
+    Column("table_owner", String),
+    Column("comment", String),
+    Column("created", DateTime),
+    Column("created_by", String),
+    Column("last_altered", DateTime),
+    Column("last_altered_by", String),
+    Column("data_source_format", String),
+    Column("storage_sub_directory", String),
+    schema="information_schema",
```

Comment:
Is it OK to use the `information_schema ` ?
From the [docs](https://docs.databricks.com/en/sql/language-manual/sql-ref-information-schema.html), I understand that it's only available when the Unity catalog is used ?

Reply:
I did some tests and it seems that, indeed, the `information_schema` is not available for non-unity workspaces.
I tried to use `SHOW TABLE EXTENDED` instead but the query seems very slow (~10s to get a result...).
So what should be done ? Fallback to STE if `information_schema` returns an error ? Or let `get_multi_table_comment` crash for non-unity workspaces (reflection will catch the error and just not return any comment) ?

Reply:
OK. So it appears other methods (`get_foreign_keys`, `get_pk_constraint`) use individual DTE statements.
So I'll do the same for table comments.
There is probably optimisations possible to try to use the `information_schema` when available and caching the results between comments/PK/FK queries but I'll leave that for potential future work.

Reply:
done